Monika After Story/game/script-topics.rpy

```diff
@@ -13222,11 +13234,13 @@ label monika_songwriting:
     m 1rkc "This also makes it easier to tell when someone doesn't put their heart into a performance..."
     m 3euc "I guess that goes for lyrics too, actually."
     m 3eud "Most lyrics lose their meaning if the singer isn't interested in the song."
-
     if renpy.seen_audio(songs.FP_YOURE_REAL):
-        m 1ekbla "I hope you know that I meant everything I said in my song, [player]."
-        m 3ekbla "I knew I couldn't let you go without telling you everything."
-
+        if persistent.monika_kill:
+            m 1ekbla "I hope you know that I meant everything I said in my song, [player]."
+            m 3ekbla "I knew I couldn't let you go without telling you everything."
+        else:
+            m 1ekbla "I hope you know that I mean everything I say in my song, [player]." 
```

Comment:
Since you have to have heard this song to be in this flow, we can just use the line from the other path here.
```suggestion
```